docs/opentelemetry/otel-mapping-summary.asciidoc

```diff
@@ -0,0 +1,969 @@
+
+The following table summarizes the alignment status by namespaces between ECS in version `9.0.0-dev`
+and OpenTelemetry semantic conventions in version `1.29.0`.
```

Comment:
do you want to track otel namespaces here to backport them later to ECS? For ECS users new namespaces might be unnecessary

Reply:
This list shows already a merge of namespaces from both, ECS and SemConv.

For example there's the `Android` namespace, which only exists in OTel but not in ECS that appears in that list

Reply:
yes, that what was my question about :) if you want to include namespaces from both standards and not only from ecs for better tracking